src/AppBundle/Tests/Controller/SubstituteControllerTest.php

```diff
@@ -41,100 +58,69 @@ public function testShow()
+        $this->assertEquals(1, $crawler->filter('td:contains("Gundersen")')->count());
 
-        // Assert that we have the edit/delete buttons as admin (1 delete button is the hidden modal button, so we have to check > 1)
-        $this->assertGreaterThan(1, $crawler->filter('a:contains("Slett")')->count());
-        $this->assertGreaterThan(0, $crawler->filter('a:contains("Rediger")')->count());
+        // Assert that we have the edit/delete buttons as admin
+        $this->assertEquals(1, $crawler->filter('button:contains("Slett")')->count());
+        $this->assertEquals(1, $crawler->filter('button:contains("Rediger")')->count());
 
-        // Team user
-        $client = static::createClient(array(), array(
-            'PHP_AUTH_USER' => 'team',
-            'PHP_AUTH_PW' => '1234',
-        ));
+        // Team member
+        $client = self::createTeamMemberClient();
 
-        $crawler = $client->request('GET', '/kontrollpanel/vikar/1');
+        $crawler = $client->request('GET', '/kontrollpanel/vikar/semester/2');
 
         // Assert that the page response status code is 200
         $this->assertEquals(200, $client->getResponse()->getStatusCode());
 
         // Assert that we have the correct page
         $this->assertEquals(1, $crawler->filter('h1:contains("Vikarer")')->count());
-        $this->assertEquals(1, $crawler->filter('td:contains("Marius")')->count());
-        $this->assertEquals(1, $crawler->filter('td:contains("Svendsen")')->count());
+        $this->assertEquals(1, $crawler->filter('td:contains("Markus")')->count());
+        $this->assertEquals(1, $crawler->filter('td:contains("Gundersen")')->count());
 
-        // Assert that we don't have the edit/delete buttons as team (1 delete button is the hidden modal button, so we have to check = 1)
-        $this->assertEquals(1, $crawler->filter('a:contains("Slett")')->count());
+        // Assert that we don't have the edit/delete buttons as team
+        $this->assertEquals(0, $crawler->filter('a:contains("Slett")')->count());
         $this->assertEquals(0, $crawler->filter('a:contains("Rediger")')->count());
-
-        // Assistant user
-        $client = static::createClient(array(), array(
-            'PHP_AUTH_USER' => 'assistent',
-            'PHP_AUTH_PW' => '1234',
-        ));
-
-        $client->request('GET', '/kontrollpanel/vikar/1');
-
-        // Assert that the page response status code is 403 access denied
-        $this->assertEquals(403, $client->getResponse()->getStatusCode());
     }
 
     public function testEdit()
     {
-        // Admin user
-        $client = static::createClient(array(), array(
-            'PHP_AUTH_USER' => 'admin',
-            'PHP_AUTH_PW' => '1234',
-        ));
+        // Team leader
+        $client = self::createTeamLeaderClient();
 
-        $crawler = $client->request('GET', '/kontrollpanel/vikar/rediger/1');
+        $crawler = $client->request('GET', '/kontrollpanel/vikar/rediger/4');
```

Comment:
Jeg får 
`[Symfony\Component\Debug\Exception\UndefinedMethodException]                             
  Attempted to call an undefined method named "getClient" of class "Symfony\Component\Dom  
  Crawler\Crawler".`
når jeg skriver `$client = $crawler->getClient();`

Det ser ikke ut til at $crawler har noen getClient() metode. Eller?